docs/5.x/cluster.md

```diff
@@ -2124,6 +2026,89 @@ $ gravity resource rm runtimeenvironment
     of runtime containers on each cluster node. Take this into account and plan each update accordingly.
 
 
+### Configuring Cluster
+
+It is possible to customize the cluster per environment before the installation or update some aspects of the cluster
+using the `ClusterConfiguration` resource:
+
+[cluster-config.yaml]
+```yaml
+kind: ClusterConfiguration
```

Comment:
I think you also forgot to add this resource to the table of all resources at the beginning of https://gravitational.com/gravity/docs/cluster/#configuring-a-cluster.